File: src/frontend/app/postavke/postavke.js
import Identity from 'auth/identity';

export default {
  postavke: null,

  getPostavke() {
    return this.postavke;
  },

  setPostavke(postavke) {
    this.postavke = postavke;
  },

  naslovSistema() {
    if (this.postavke)
      return this.postavke.naslovSistema;
    return 'Project Management System';
  },

  trajanjeSesije() {
    if (this.postavke)
      return this.postavke.trajanjeSesije;
    return 5;
  },

  urlKarte() {
    if (this.postavke)
      return this.postavke.urlKarte;
    return 'http://{s}.tile.osm.org/{z}/{x}/{y}.png';
  },

  autorskaPravaKarte() {
    if (this.postavke)
      return this.postavke.autorskaPravaKarte;
    return '';
  }

};

File: src/backend/Web/Web.Entities/Models/Base/Postavke.cs
using System;
using System.Collections.Generic;
using System.ComponentModel.DataAnnotations;
using System.ComponentModel.DataAnnotations.Schema;
using System.Text;

namespace Web.Entities.Models.Base
{
    /// <summary>
    /// Postavke sistema
    /// </summary>
    public class Postavke : EntityAutoriziran
    {
        /// <summary>
        /// Konstruktor sa defaultnim vrijednostima postavki
        /// </summary>
        public Postavke()
        {
            NaslovSistema = "Web";
            TrajanjeSesije = 5;
            UrlKarte = "http://{s}.tile.osm.org/{z}/{x}/{y}.png";
            AutorskaPravaKarte = @"&copy; <a href=""http://osm.org/copyright"">OpenStreetMap</a> contributors";
           
        }

        /// <summary>
        /// Id postavki
        /// </summary>
        [Key, Column(Order = 0)]
        [DatabaseGenerated(DatabaseGeneratedOption.Identity)]
        public int Id { get; set; }

        /// <summary>
        /// Naslov sistema (koristi se u frontendu)
        /// </summary>
        [Required]
        [StringLength(64)]
        public String NaslovSistema { get; set; }

        /// <summary>
        /// Trajanje sesije korisnika u danima
        /// </summary>
        [Range(1, 10)]
        public int TrajanjeSesije { get; set; }

        /// <summary>
        /// Url koji se koristi kao provajder karti
        /// </summary>
        [Required]
        [StringLength(128)]
        public String UrlKarte { get; set; }

        /// <summary>
        /// Autorska prava provajdera karte
        /// </summary>
        [Required]
        [StringLength(256)]
        public String AutorskaPravaKarte { get; set; }

     
    }
}
